go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/vmware-tanzu/sonobuoy
 
-go 1.23
+go 1.23.0
```

Comment:
I remember there is a golang related CVE on 1.23.0, could you update this to 1.23.1? Thanks

Reply:
Thank you for comment. I updated it to 1.23.2.